library/core/src/option.rs

```diff
@@ -1125,6 +1125,7 @@ impl<T> Option<T> {
     /// ```
     #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[must_use = "if you don't need the result, use `if let` instead"]
```

Comment:
Part of my mind wants to imagine this should be
```suggestion
    #[must_use = "if you don't need the option, use `if let` instead"]
```
but I don't think that's actually what you meant here.

Reply:
I actually written what I meant — "the result [of this operation/function]". Although I see how that could be confusing, mhh...

Would ``"if you don't need the return value of this function, use `if let` instead"`` be better?

Reply:
Sure!
or just
```rust
#[must_use = "if you don't need the returned value, use `if let` instead"]
```

Reply:
I like your option better (pun intended) :3